### ISSUE TITLE
Check Kaki Damage

### ISSUE THREAD

**Issue by claytondaley** [Skibisky/RuneManager#89]
Need to verify that "no crit" is handled correctly.  For SKOGUL, etc. this is FIXED which may work correctly (or not!)

**Comment by claytondaley** [comment]
Ideally, we'd also support his ATK => DEF => DMG passive.

**Comment by Skibisky** [comment]
Sadly the passive has no information that I could use to work that in.

I'm unsure about manually specifying rules for individual monsters damage, but that might be a system to look into later.

**Comment by claytondaley** [comment]
Good Call.  I hoped they included the FIXED tag in the formula which is how Mo/Skogul don't crit.  Probably need a separate configuration file that applies some post-load fixes like this.  Could also be a place to e.g. modify default thresholds like I mention in #65.

**Comment by claytondaley** [comment]
We had some discussion over on the Swarfarm discord that FIXED != No Crit.  Perhaps an attack buff would be an example of something that affects Kaki (no crit) but not Mo Long (fixed).
